content/_startups/itou.md

```diff
@@ -1,19 +1,59 @@
 
-La Startup d'Etat "Portail de l'inclusion" a pour objectif de développer et faire vivre une plate-forme numérique afin de simplifier la vie des acteurs de l’inclusion (IAE, EA…), de renforcer les capacités de coopération, d’innovation et d’accompagnement social et professionnel du secteur et de mieux évaluer l’impact social et les moyens affectés.
+Aujourd'hui **[inclusion.beta.gouv.fr](https://inclusion.beta.gouv.fr/)** permet de mettre en relation des chercheurs d'emploi (aussi nommés candidats) auprès d'[employeurs solidaires](https://doc.inclusion.beta.gouv.fr/pourquoi-cette-plateforme/les-acteurs/employeurs-solidaires) (entreprises, associations...). Les candidats peuvent également bénéficier de l'aide de [prescripteurs](https://doc.inclusion.beta.gouv.fr/pourquoi-cette-plateforme/les-acteurs/prescripteurs-habilites) (Pôle emploi, Missions locales, Cap Emploi...).
+
+### Contexte
+
+L’**insertion par l’activité économique** (IAE) est un dispositif dans lequel l’État
+confie une mission d’accompagnement socio-professionnel de personnes
+en difficulté à plus de 3 700 strucutres associatives ou privées. Ce dispositif bénéficie à des publics très éloignés de l’emploi (134 000 personnes en parcours d’insertion fin 2017). Il repose sur l’association de trois facteurs-clés : une mise en situation professionnelle, un accompagnement professionnel mais aussi social, et la mobilisation d’actions de formation. 
+
+L'**inclusion dans l'emploi** concerne le vaste dispositif de l'IAE, les [entreprises adaptées](https://travail-emploi.gouv.fr/emploi/emploi-et-handicap/article/emploi-et-handicap-les-entreprises-adaptees-ea) (EA) dont la spécificité est d'employer au moins 80% de travailleurs en situation de handicap, et les [groupements d’employeurs pour l’insertion et la qualification](https://travail-emploi.gouv.fr/emploi/insertion-activite-economique/article/groupements-d-employeurs-pour-l-insertion-et-la-qualification-geiq) (GEIQ).
+
+### Problématique
+
+Plus de 500.000 personnes sont en chômage de très longue durée (2 an ou plus) ([source INSEE](https://www.insee.fr/fr/statistiques/2417491#tableau-figure1)). L'Insertion par l’Activité Economique (IAE) peut être une des solutions, seulement un certain nombre de problèmes ont été soulevés par la [Cour des Comptes](https://www.ccomptes.fr/system/files/2019-01/20190115-rapport-insertion-chomeurs.pdf), nous alertant sur des freins d’accès à l’IAE et des modèles économiques, peu connus, aux équilibres fragiles. 
+
+Actuellement il existe des délais entre l’orientation d’une personne et son début de contrat. L'agrément IAE, définissant l'éligibilité d'une personne à l'IAE, doit être fiabilisé et simplifié. Enfin, pour les organisations, de nombreux protocoles de gestion sont fastidieux et complexes avec l’[Agence de Services et de Paiement](https://www.asp-public.fr/) (ASP) et les [Direccte](http://direccte.gouv.fr/).
```

Comment:
Attention, ça peut mettre l'équipe en porte-à-faux vis à vis de l'ASP.  Je resterais sur le pb : 

"les entreprises et autres acteurs de l'IAE passent un temps très important à saisir des données ou établir des rapports sur leur activité, charge administrative récurrente et parfois redondante d'un rapport à l'autre "

Reply:
ok noté